docs/topics/exceptions.rst

```diff
@@ -77,3 +78,31 @@ NotSupported
 
 This exception is raised to indicate an unsupported feature.
 
+StopDownload
+-------------
+
+.. versionadded:: 2.2
+
+.. exception:: StopDownload(fail=True)
+
+Raised from a :class:`~scrapy.signals.bytes_received` signal handler to
+indicate that no further bytes should be downloaded for a response.
+
+The ``fail`` boolean parameter controls which method will handle the resulting
+response:
+
+* If ``fail=True``, the request errback is called. The response object is
+  available as the ``response`` attribute of the received
+  :class:`~twisted.python.failure.Failure` object. This is the default behaviour.
+
+* If ``fail=False``, the request callback is called instead.
+
+Note that ``fail`` is a keyword-only parameter.
+
+In both cases, the response could have its body truncated: the body contains
```

Comment:
What do you think about moving a note about kwarg-only argument somewhere else, to not separate 2 cases from "In both cases, ..."?